scripts/release.sh

```diff
@@ -30,7 +30,7 @@ fi
 
 # Fix until we can update this script to follow semantic versioning
 if [[ $WORKFLOW_NAME == amr ]]; then
-    TAG="amr-v1.3.0"
+    TAG="amr-v1.3.1"
```

Comment:
If we remove this branch and do a `patch` release from main, will we be able to get this tag?

Reply:
No, because this release script doesn't distinguish between beta/release candidates and full versions - it's not fully compliant with semantic versioning, unfortunately. I am hoping we can fix it in the then near future.